### PR TITLE
(IMAGES-726) Add template for Fedora 27 x86_64

### PR DESCRIPTION
**Pull request by ScottGarman**
I'm able to build the vmware base template with this PR.

### CODE REVIEW

**Comment by ScottGarman** [comment]
@ekinanp good catch, both of those manifests are relevant for the vsphere template and I've now updated them accordingly.

File: templates/linux/fedora/27/x86_64/vars.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "template_name"                         : "fedora-27-x86_64",
+    "template_os"                           : "fedora-64",
+    "beakerhost"                            : "fedora27-64",
+    "version"                               : "0.0.1",
+    "iso_url"                               : "http://osmirror.delivery.puppetlabs.net/iso/Fedora-Server-dvd-x86_64-27-1.6.iso",
+    "iso_checksum"                          : "e383dd414bb57231b20cbed11c4953cac71785f7d4f5990b0df5ad534a0ba95c",
+    "iso_checksum_type"                     : "sha256",
+    "vmware_vsphere_nocm_vmx_data_memsize"  : "4096",
+    "vmware_vsphere_nocm_vmx_data_numvcpus" : "2",
+    "puppet_aio"                            : "http://yum.puppetlabs.com/fedora/f26/PC1/x86_64/puppet-agent-1.10.9-1.fedoraf26.x86_64.rpm"
```
Comment (branan) [inline templates/linux/fedora/27/x86_64/vars.json]: Am I correct that we're just using F26 for bootstrap and will switch this once we have a F27 build?

Comment (ScottGarman) [reply templates/linux/fedora/27/x86_64/vars.json]: @branan yep, that's the workflow we use to workaround the chicken-and-egg problem

**Comment by ScottGarman** [comment]
@ekinanp Building the vsphere template currently fails because it tries to use a local mirror of the Fedora 27 repository that we don't have set up yet. So until we get that set up, I won't be able to validate that further.

**Comment by ekinanp** [comment]
Oh ok, thanks for the clarification @ScottGarman

**Comment by shrug** [comment]
We should set up a F27 mirror in Artifactory...

**Comment by shrug** [comment]
tada: https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote__fedora/27/

**Comment by ScottGarman** [comment]
@shrug I'm not sure if that's working correctly yet - for example, http://osmirror.delivery.puppetlabs.net/fedora27-x86_64/ still returns a 404.

**Comment by shrug** [comment]
Right, I put it in Artifactory instead of osmirror: https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote__fedora/

**Comment by ScottGarman** [comment]
Ok, adding a don't merge on this until I get the artifactory mirror urls working - may as well enable them for all of our Fedora releases at once.

**Comment by shrug** [comment]
Oh, the url has changed (I made a typo orignally) - it's now https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_fedora/ (only one underscore between "remote" and "fedora"

**Comment by ScottGarman** [comment]
@shrug Is there an "updates" or "everything" mirror to include anymore? All I'm seeing is the "os" one. Previously we set up local repositories for all three.

**Comment by shrug** [comment]
@ScottGarman Oops, we mirrored a level too low to catch the Updates repos. Take a look at https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_fedora/ - you should see everything?

**Comment by shrug** [comment]
ping @ScottGarman Have you been able to test with the change above?

**Comment by ScottGarman** [comment]
I've been on PTO, returning tomorrow and hope to try this out soon.

**Comment by ScottGarman** [comment]
I've now been able to use this to build both the base and vsphere templates, the artifactory repo appears to be working for me with the included tweaks.

**Pull request merged**